chia/types/clvm_cost.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from chia.util.ints import uint64
+
+
+@dataclass
+class CLVMCost:
```

Comment:
Yes, I think you are right.

btw,  I don't think it would be safe to redefine FeeRate this same way, because I want to enforce that a FeeRate is constructed from a CLVMCost and a Mojos.

Reply:
Done